packages/golden-test-generator/src/Block/getBlocks.ts

```diff
@@ -39,7 +40,8 @@ export const getBlocks = async (
           compactGenesis: await StateQuery.genesisConfig(
             await createInteractionContext(reject, logger.info, { connection: options.ogmiosConnectionConfig })
           ),
-          intersection: undefined
+          // Review: this can't be undefined acccording to type
+          intersection: undefined as unknown as Intersection
```

Comment:
```suggestion
          intersection: undefined as unknown as ChainSync.Intersection
```

Reply:
I don't like the double cast, the expected type is non-nullable so it's a potential bug when using this